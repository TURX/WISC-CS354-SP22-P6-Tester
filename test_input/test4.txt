0 64 16