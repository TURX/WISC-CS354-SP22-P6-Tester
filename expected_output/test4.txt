Reading character starting at index 0 to 64 with an interval of 16

Reading character at index 0   : data = A : hit count = 0   : miss count = 1   : read data count = 1
Reading character at index 16  : data = e : hit count = 1   : miss count = 1   : read data count = 1
Reading character at index 32  : data =
 : hit count = 1   : miss count = 2   : read data count = 2
Reading character at index 48  : data =   : hit count = 2   : miss count = 2   : read data count = 2
